llms/README.md

```diff
@@ -61,7 +61,7 @@ text using the given prompt.
 For a full list of options run:
 
 ```
-python -m mlx_lm generate --help
+python -m mlx_lm.generate --help
```

Comment:
Small typo I found while working on updating the phi2 model args.

Reply:
👍